gatsby/src/docs/accounts/sso/azure-sso.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Azure Active Directory SSO'
+sidebar_order: 1
+---
+
+## Installation
+
+1. Login to the [Azure portal](https://portal.azure.com/).
+
+2. (You can skip to (3) if you don't need to create a permission group for Sentry.) Under 
+Manage Azure Active Directory (the shield picture), click "View", which takes
+you to the Organization Overview page. On the left hand navigation, click "Groups". Create a 
+new group, add an owner, and members to the group. Set "Group type" to Office 365. For more 
+details about group creation, see the
+[Azure docs](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-groups-create-azure-portal).
+
+3. On the left hand navigation, click "Groups". Create a new group, add an owner, and
+members to the group. Set "Group type" to Office 365. For more details about group creation, see the
+[Azure docs](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-groups-create-azure-portal).
+
+4. Return to the Overview page. On the left hand navigation, click "Enterprise Applications".
+Then, select "Non-gallery application" and create one for Sentry. Once created, you'll
+be directed to "Sentry - Overview."
+
+5. Click on `1. Assign users and groups`, and assign yourself and the group you've created 
+to the Sentry app.
+
+6. Click on `2. Setup Single Sign-on`
+
+7. For Section (1) labeled as `Basic SAML2 Configuration`, enter the following data 
+in each line.
+
+    
+    Identifier (entity ID): https://sentry.io/saml/metadata/sentry/
+
+    Reply URL (Assertion Consumer Service URL): https://sentry.io/saml/acs/sentry/
+
+    Sign on URL: https://sentry.io/auth/login/sentry/
+
+    Relay State: https://sentry.io/
+
+    Logout URL: https://sentry.io/saml/sls/sentry/
+    
+8. In Section (3) labeled as `SAML Signing Certificate`, copy the `App Federation Metadata URL`
+
+[ screenshot here ]
+
+9. Go to `https://sentry.io/settings/<YOUR-ORGANIZATION-SLUG>/auth/` and click on "Configure" 
+for Active Directory
+
+10. Paste the App Federation Metadata URL and click "Get metadata"
+
+11. In the following page, enter these to map the values from AzureAD to Sentry.
+
+    
+    IdP User ID: http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name
+    
+    User Email: http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress
+    
+    First Name: http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname
+    
+    Last Name: http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname
+    
+For more details about mappings for custom configs, see 
+[The Role of Claims](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/technical-reference/the-role-of-claims).
+
+12. Congrats. You've set up an intance of AzureAD with Sentry.
```

Comment:
This isn't a step. Keep it as plain text (or remove it if it's not consistent with our other Integrations doc).